docs/rules/README.md

```diff
@@ -276,42 +277,42 @@ The following rules extend the rules provided by ESLint itself and apply them to
 
 | Rule ID | Description |    |    |
 |:--------|:------------|:--:|:--:|
-| [vue/array-bracket-newline](./array-bracket-newline.md) | enforce linebreaks after opening and before closing array brackets in `<template>` | :wrench: | :lipstick: |
```

Comment:
`git pull origin master && npm run update` should probably fix these unrelated changes.

Reply:
complete